otter/convergence/planning.py

```diff
@@ -186,29 +186,22 @@ def _converge_lb_state(server, current_lb_nodes):
         if desired.equivalent_definition(node.description)]
 
     if desired_matching_existing:
-        met_desireds, good_nodes = zip(*desired_matching_existing)
```

Comment:
Honestly I don't like the unzipping functionality in `zip` function. There should've been `unzip` for that. But I can understand the discomfort with `_`. Will change it :)